packages/app-store/zoho-bigin/lib/CalendarService.ts

```diff
@@ -83,31 +83,35 @@ export default class BiginCalendarService implements Calendar {
     };
 
     const tokenInfo = await refreshOAuthTokens(
-      async () =>
-        await axios.post(accountsUrl, qs.stringify(formData), {
+      async () => {
+        const response = await fetch(accountsUrl, {
```

Comment:
Axios has the `data` property but not the `body`. To ensure compatibility with credential syncing, let's use the native fetch method.